e2e/image/build_test.go

```diff
@@ -37,7 +37,12 @@ func TestBuildFromContextDirectoryWithTag(t *testing.T) {
 		withWorkingDir(dir))
 	defer icmd.RunCommand("docker", "image", "rm", "myimage")
 
-	result.Assert(t, icmd.Expected{Err: "DEPRECATED: The legacy builder is deprecated and will be removed in a future release."})
+	const buildxMissingWarning = `DEPRECATED: The legacy builder is deprecated and will be removed in a future release.
```

Comment:
Haven't run the e2e tests, so they may fail here (whitespacing and such)

Reply:
Yes looks like this change can be reverted now that the warning message has been removed from build pkg.

```
=== FAIL: e2e/image TestBuildFromContextDirectoryWithTag (2.81s)
    build_test.go:45: assertion failed: 
        Command:  docker build -t myimage .
        ExitCode: 0
        Stdout:   Sending build context to Docker daemon  4.608kB

        Step 1/4 : FROM registry:5000/alpine:3.6
         ---> 76da55c8019d
        Step 2/4 : COPY run /usr/bin/run
         ---> 485496f425b9
        Step 3/4 : RUN run
         ---> Running in 48dd9744aa87
        running
        Removing intermediate container 48dd9744aa87
         ---> 076deca8990c
        Step 4/4 : COPY data /data
         ---> 0ee25cd5a8cb
        Successfully built 0ee25cd5a8cb
        Successfully tagged myimage:latest
        
        Stderr:   
        
        Failures:
        Expected stderr to contain "DEPRECATED: The legacy builder is deprecated and will be removed in a future release.\n            Install the buildx component to build images with BuildKit:\n            https://docs.docker.com/go/buildx/\n"
```

But in this test it should display a warning about the deprecated legacy builder and it doesn't seem to pop out.